Classes/Classes/miniphonedir.cpp

```diff
@@ -0,0 +1,101 @@
+#include<iostream> 
+
+using namespace std; 
+
+  
+
+class UserData { 
+
+private: 
+
+    string name; 
+
+    string last_name; 
+
+    string email; 
+
+    string phone_number; 
+
+    /*Method*/ 
+
+public: 
+
+    void set_name(string name_temp); 
+
+    string get_name(); 
+
+    
+
+     
+
+}; 
+
+  
+
+void UserData::set_name(string name_temp) 
+
+{ 
+
+    name = name_temp; 
+
+} 
+
+  
+
+string UserData::get_name() 
+
+{ 
+
+    return name; 
+
+} 
+
+  
+
+int main(void) 
+
+{ 
+
+    int option; 
+
+    string name; 
+
+    do { 
+
+        cout << "Directorio Telefonico\n"; 
+
+        cout << "Seleccione una opcion:\n\n"; 
+
+        cout << "1. Agregar Nuevo\n\n"; 
+
+        cout << "Seleccione:"; 
+
+        cin >> option; 
+
+  
+
+        switch (option) 
+
+        { 
+
+        case 1:  
+
+            system("cls"); 
+
+            UserData User_New; 
+
+            cout << "Nombre:"; 
```

Comment:
Add "\t" for separate the word cout of new word cin e.g.

In execution with out "\t" 
-------------------------------------------
Directorio Telefonico
Seleccione una opcion:

1. Agregar Nuevo

**_Seleccione:1_**
**_Nombre:Juan_**
JuanDirectorio Telefonico

In execution with "\t" 
-------------------------------------------
Directorio Telefonico
Seleccione una opcion:

1. Agregar Nuevo

**_Seleccione:______1_**
**_Nombre:_________Juan_**
El nombre ingresado es: Juan

Directorio Telefonico